config/default/common/config/metadata/stories/sunglint/step002.md

```diff
@@ -1 +1 @@
-<p>You can see these sunglint areas roughly in the center of each of these MODIS/Terra swaths. You may also be wondering what the black areas of missing imagery near the equator are too! This occurs with MODIS/Terra and MODIS/Aqua imagery because the swath width is 2,330km wide.</p>
+<p>You can see these sunglint areas in water bodies roughly in the center of each of these Moderate Resolution Imaging Spectroradiometer (MODIS) sensor/Terra satellite swaths. You may also be wondering what the black areas of missing imagery near the equator are too! This occurs with MODIS/Terra and MODIS/Aqua imagery because the swath width is only 2,330km wide.</p>
```

Comment:
Even though the next step more fully explains why there's a gap, I think this step may need a bit more explanation at the end, e.g., "...which isn't enough to overlap the previous swath."